.gitignore

```diff
@@ -4,7 +4,7 @@
 /node_modules/
 /.pnp
 .pnp.js
-package.lock.json
+package-lock.json
```

Comment:
@chiraag-kakar Pod leader mentioned in last meeting that it would be best to commit package-lock.json along with other files, so could you please remove this from .gitignore?